lib/twitter_cldr/js/renderers/numbers/rbnf/rule_group_renderer.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module Numbers
+
+        class RBNFRuleGroupRenderer < TwitterCldr::Js::Renderers::Base
```

Comment:
Including my comment above about having a nesting `RBNF` module defined in `numbers/rbnf.rb`, I think you could just dump all these small one-line renderers that just set the template into the same file instead of having them all in separate files. Though, I'd still leave anything bigger (e.g., `RBNFTestHelperRenderer`) in separate files.